FreeRTOS/Demo/readme.txt

```diff
@@ -17,3 +17,9 @@ the WinAVR tools then the AVR_ATMega323_WinAVR directory contains the
 relevant makefile.  The makefile includes files from the Demo/ATMega323 
 and the Demo/Common directories.  If this is the only port you are 
 interested in then all the other directories can be ignored.
+
+Community Supported Demos

```

Comment:
Did you consider adding this in the main readme itself?  @n9wxu

Reply:
An off topic(?) point: I would have like this to be a .md file so that github could render the text in a more readable manner. The "community supported demos" line should be a highlighted (or a heading) section to attract user's attention to it.

Reply:
Maybe not so off topic - I propose to make it an md file.

Reply:
@AniruddhaKanhere I also like the md file.  I did not use an md file because the current file is txt and it would want additional formatting to make a pretty md.